src/highdicom/seg/sop.py

```diff
@@ -893,6 +850,62 @@ def add_segments(
         if len(self.PixelData) % 2 == 1:
             self.PixelData += b'0'
 
+    def _format_pixel_array(
```

Comment:
Is this supposed to return `np.ndarray`?

Reply:
Yes, it's supposed to. Fixed with 3badeb3c2910721dd2dc6dbe1fc7cf35bded1787